lib/ansible/modules/net_tools/dnsmadeeasy.py

```diff
@@ -560,7 +559,7 @@ def main():
             record_value=dict(required=False),
             record_ttl=dict(required=False, default=1800, type='int'),
             monitor=dict(default='no', type='bool'),
-            systemDescription=dict(default=''),
+            systemDescription=dict(),
```

Comment:
```suggestion
            systemDescription=dict(required=True),
```
Otherwise it's not really required.

Reply:
systemDescription is only required if monitor is true, which is stated in the required_if.

Reply:
Then it should not say that it is always required in the documentation (which it currently still does).

Reply:
Missed the documentation change. Updated now.